test.py

```diff
@@ -1,4 +1,4 @@
 a = int(input('Enter num 1 : '))
 b = int(input('Enter num 2 : '))
 
-print((a+b))
```

Comment:
Semi colon is not needed in Python files. Though it's good to have but not mandatory